IGNITE-23168 Event consumption time metric in CDC consumer (#11612)
apache/ignite: docs/_docs/persistence/change-data-capture.adoc

| BinaryMetaDir | Binary meta-directory the application reads data from.
| MarshallerDir | Marshaller directory the application reads data from.
| CdcDir | The CDC directory the application reads data from.

> | WalProcessingTime | Time of WAL segment processing, in milliseconds.

Reviewer: SegmentConsumingTime

Author: Done!

| BinaryMetaDir | Binary meta-directory the application reads data from.
| MarshallerDir | Marshaller directory the application reads data from.
| CdcDir | The CDC directory the application reads data from.

> | EventsConsumptionTime | Time for which this CDC client processed the WAL segment events, in nanoseconds.

Reviewer 2: ```suggestion
| EventsConsumptionTime | Time of WAL segment events processing by a CDC consumer, in nanoseconds.
```

Author: Agree

> | EventsConsumptionTimeTotal | The total time for which this CDC client processed the WAL segments events, in nanoseconds.

Reviewer 2: ```suggestion
| EventsConsumptionTimeTotal | Total time of events processing by a CDC consumer, in nanoseconds.
```

Author: Agree
